data/validateQuestionLink.js

```diff
@@ -0,0 +1,80 @@
+import { questions } from "./question-data.js";
+import fs from "fs/promises";
+import path from "path";
+
+// Run this script and check logs/broken-links-logs.csv for broken links
+
+const STATUS_SUCCESS = 200;
+const LOGS_PATH = "../logs/broken-links-logs.csv";
+
+// The input param is the link to be validated on.
+export const validateURLResponse = async (input) => {
+  try {
+    const response = await fetch(input);
+
+    const status = response.status;
+
+    return status === STATUS_SUCCESS;
+  } catch (error) {
+    console.error(
+      `An error occured while accessing "${input}". \nError: ${error}`,
+    );
+    return false;
+  }
+};
+
+const validateQuizLink = async (linkList) => {
+  console.info("validating links...", linkList.length);
+  const badLinks = [];
```

Comment:
Maybe we can assign `badLinks` to the `Promise.all` and then retrieve the links in the return.

Reply:
fixed